src/test/java/com/make/my/day/hm1/Homework01.java

```diff
@@ -117,7 +117,8 @@ public void transformAndProvideSumWithCounter() {
   }
 
   @Test
-  public void sortByNameLength() {
+  public void sortNamesAlphabetically () {
```

Comment:
Nope, here we sort by name length

Alphabetically if we have "Alfred", "Bill", "Cavin" etc

Reply:
I must have been missed the changes in original branch.